unit/memleak_hawaii3.py

```diff
@@ -39,13 +39,23 @@
     ax.pcolor(10 * rand(50, 50))
 
     fig.savefig('tmp%d' % i, dpi=75)
-    close(1)
+    plt.close(1)
 
-    gc.collect()
+    coll = gc.collect()
     val = report_memory(i)
     print(i, val)
     if i == indStart:
         start = val  # wait a few cycles for memory usage to stabilize
+    mem_size.append(val)
+    coll_count.append(coll)
 
 end = val
-print('Average memory consumed per loop: %1.4fk bytes\n' % ((end - start) / float(indEnd - indStart)))
+print('Average memory consumed per loop: %1.4fk bytes\n' %
+      ((end - start) / float(indEnd - indStart)))
+fig, ax = plt.subplots()
+ax2 = ax.twinx()
+ax.plot(mem_size, 'r')
+ax.set_ylabel('memory size', color='r')
+ax2.plot(coll_count, 'k')
+ax2.set_ylabel('collect count', color='k')
+fig.savefig('report')
```

Comment:
Just more information.  I was checking that the memory increase is uniform (which it is :disappointed: )

I don't think it is all in one place either.  Removing all of the plotting calls, and just making, saving, and closing the figures/axes results in a 3k/loop leak.

I am leaning towards this not blocking the release.